docs/development/docs-contribute.md

```diff
@@ -169,6 +169,32 @@ HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])
 HUMAN_READABLE_BINARY_BYTE_FORMAT(value, \[precision])
 :::
 
+#### Markdown table format
+
+When editing or adding tables, do not include extra characters to "prettify" the table format within the Markdown source.
+Some code editors may format tables by default.
+For an example of how to disable it, see the [IntelliJ formatting file for Druid](https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L78).
```

Comment:
Or it'd be better to import the code style file - https://github.com/apache/druid/blob/master/dev/style-conventions.md